packages/rancher-vsphere-cpi/package.yaml

```
@@ -1,4 +1,4 @@
 url: https://github.com/rancher/vsphere-charts.git
 subdirectory: charts/rancher-vsphere-cpi
 commit: 81da645ddf86370fc5d1e35dc4711a45b3cea929
-version: 100.2.0
+version: 100.2.1
```

For the CI failure - I copied the package.yaml from rancher charts, but it looks like the version is actually ignored, and we want to do this instead:
```suggestion
packageVersion: 01
```

---

Ah the old scripts

---

AFAIK packageVersion is a Rancher 2.5 thing, but my understanding is that rke2-charts doesn't have the latest charts build scripts so it may be old enough that it still uses packageVersion. This needs to be checked.